tutorial/11_qt/README.rst

```diff
@@ -16,7 +16,7 @@ pyinstaller and the Qt framework. (15 min for talk, 10 for exercise)
 Using PyVista with QT
 ---------------------
 
-PyVista has an interface for placing plots in `QT <https://www.qt.io/>` in the
+PyVista has an interface for placing plots in `QT <https://www.qt.io/>`_ in the
```

Comment:
```suggestion
PyVista has an interface for placing plots in QT `(see QT) <https://www.qt.io/>`_ in the
```

Reply:
reference needs to be unique or we need to define the reference elsewhere to use in all places